suricata.yaml.in

```diff
@@ -560,12 +560,30 @@ af-packet:
     #  Requires at least Linux 3.14.
     #  * cluster_rollover: kernel rotates between sockets filling each socket before moving
     #  to the next. Requires at least Linux 3.10.
+    #  * cluster_ebpf: kernel rotates between sockets following an extended BPF function loaded
+    #  from a file. User needs to use ebpf-lb-file option to set the path to the file
+    #  to load. Requires at least Linux 4.2.
     # Recommended modes are cluster_flow on most boxes and cluster_cpu or cluster_qm on system
     # with capture card using RSS (require cpu affinity tuning and system irq tuning)
     cluster-type: cluster_flow
     # In some fragmentation case, the hash can not be computed. If "defrag" is set
     # to yes, the kernel will do the needed defragmentation before sending the packets.
     defrag: yes
+    # eBPF file containing a 'loadbalancer' function that will be inserted into the
+    # kernel and used as load balancing function
+    #ebpf-lb-file:  @e_sysconfdir@lb.bpf
```

Comment:
this should probably not be in the config dir, instead in /usr/lib/suricata or something?